setup.py

```diff
@@ -39,14 +39,13 @@ def read(*paths):
     cmdclass={'build_py': build_py},
     version=VERSION,
     description='Clearbit python bindings',
-    long_description=read('README.rst'),
+    long_description=read('README.md'),
     author='Clearbit',
     author_email='support@clearbit.com',
     url='https://clearbit.com',
     packages=['clearbit', 'clearbit.enrichment'],
     package_data={'clearbit': ['../VERSION']},
     install_requires=install_requires,
-    use_2to3=True,
```

Comment:
`nitpick:` I guess this could be the reason why the imports didn't work

`Long details:`
So the [`from prospector import Prospector`](https://github.com/15five/clearbit-python/blob/master/clearbit/__init__.py#L9) would work perfectly fine in `python2.*` environment, because imports work differently there and it would actually import the module
The reason it worked within python3 as well, was this line, since it most probably converted the imports or adopted them to py3
Again just wanted to leave it for the info, it will be fixed anyways with https://github.com/15five/clearbit-python/pull/2